config.json.template

```diff
@@ -0,0 +1,11 @@
+{
+  "app_name": "TD Five Boro Bike Tour",
+  "dcs_url": "SERVER_PUBLIC_URL/DNS/IP",
+  "tour_id": "ACTIVE_TOUR_ID",
+  "tour_start_beta_time": 0,
+  "tour_end_beta_time": 1,
```

Comment:
Should these really be 0 and 1? Granted we never used them, it might be a good idea to comment 'hey these don't really need to be used' or something??